src/utils/abiManager.js

```diff
@@ -0,0 +1,40 @@
+class ABIManager{
+  chainId = null;
+
+  constructor(chainId){
+      this.chainId = chainId;
+  }
+  
+  getABIDirectory(){
+    const ABIDirectory = __dirname +  '/../abis';
```

Comment:
@forcey having a hard time getting this path to resolve correctly, although something like this work with the deploy script, when calling it in the browser it wont resolve the file correctly.

Reply:
try "../abis"?

Reply:
i must have tried every variation multiple times.

MFW

![msvck4E4zE1qz4rgp](https://user-images.githubusercontent.com/1947534/150733370-7ecedaed-7f7c-457f-b663-382fe9a08a72.jpg)